CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Added
+
+- Add a `WithStatus` option to `go.opentelemetry.io/otel/trace` to allow setting the span status when calling `span.RecordError`. (#1677)
```

Comment:
I think saying `to the trace package` should be enough here.

Reply:
Please keep the full package name `go.opentelemetry.io/otel/trace`. It addresses the ambiguity of which trace package is being discussed and provides new users who are searching the docs a universality recognized package name.